build.gradle

```diff
@@ -32,7 +31,7 @@ ext {
 allprojects {
     repositories {
         google()
-        jcenter()
```

Comment:
Why is this changing?

Reply:
Yeah apparently this isn't just a cardinal thing. We got some gradle errors the other day because of a scheduled brown out. We may need to do this in all the repos.